cmd/pulumi-test-language/interface.go

```diff
@@ -841,12 +841,65 @@ func (eng *languageTestServer) RunLanguageTest(
 			main,
 			project.Runtime.Options())
 
-		// TODO(https://github.com/pulumi/pulumi/issues/13941): We don't capture stdout/stderr from the language
-		// plugin, so we can't show it back to the test.
-		err = languageClient.InstallDependencies(plugin.InstallDependenciesRequest{Info: programInfo})
+		installStdout, installStderr, installDone, err := languageClient.InstallDependencies(
+			plugin.InstallDependenciesRequest{Info: programInfo},
+		)
 		if err != nil {
 			return makeTestResponse(fmt.Sprintf("install dependencies: %v", err)), nil
 		}
+
+		// We'll use a WaitGroup to wait for the stdout (1) and stderr (2) readers to be fully drained, as well as for the
+		// done channel to close (3), before we carry on.
+		var wg sync.WaitGroup
```

Comment:
https://pkg.go.dev/golang.org/x/sync/errgroup might be a good fit for this

Reply:
Annoyingly that only returns the first error, whereas I opted here to do `errors.Join`. That might be unnecessarily cautious on my part, so happy to switch to `errgroup` if you feel better?

Reply:
I think this is fine, helpful to have more error context if things do break